code/segmentedtickerplant/sctp.q

```diff
@@ -50,6 +50,6 @@ init:{
 
 // Extract data from incoming table as a list
 upd:{[t;x]
-  x:flip value each x;
```

Comment:
Is this based on Jonny's comment from earlier?

Reply:
yes, sorry unrelated to the logging corrections

Reply:
haha no problem!